bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js

```diff
@@ -1713,7 +1731,151 @@ var WebApp = (function() {
 			}
 		}
 	}
-
+	
+	function Slider(item) {
+		var _t = this;
```

Comment:
Maybe `that`. The underscore confused me at first sight. `(function(a, t)...` is another example. And some comments for functions would be nice. But you're right, the rest of `Slider` is much more readable that the original Logic.js.